index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Hello World, we are software developers!</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="index.js"></script>
+    <nav id="navbar">
+      <a class="navlink" href="#welcome-section">About</a>
+      <a class="navlink" href="#projects">Work</a>
+      <a class="navlink" href="#profile">Profile</a>
+    </nav>
+    <div id="welcome-section">
+      <h1>Hey, i am Favour</h1>
+      <p class="wsp"><em>a software developer</em></p>
+    </div>
```

Comment:
I think it is best practice to put your `nav` and `welcome-section` in a `header tag`.  The header contains introductory information about the website and it an important part of the web page that defines the structure of our content.

You can refactor your code to look something like this 👇🏽 

```
  <header>
      <nav id="navbar">
        <a class="navlink" href="#welcome-section">About</a>
        <a class="navlink" href="#projects">Work</a>
        <a class="navlink" href="#profile">Profile</a>
      </nav>
      <div id="welcome-section">
        <h1>Hey, i am Favour</h1>
        <p class="wsp"><em>a software developer</em></p>
      </div>
   </header>
```

What do you think about refactoring your code to look like the code snippet I have added above? With this, you would have followed HTML's best semantics.

Reply:
- [ ] According to the [step-by-step guide of giving a Code Review](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate), it is always good to explain **why** you request a change.
    I think it will be better if you add the reason of this suggestion - it might be obvious for you but not for someone else. :no_good:

Reply:
- [ ] Please Remember to state reasons why it should be in a header tag as suggested,
- [ ]  [optional] it would be better to add some code snippet to explain further